src/Compilers/CSharp/Portable/Binder/BinderFactory.BinderFactoryVisitor.cs

```diff
@@ -1401,7 +1401,12 @@ internal static MemberDeclarationSyntax GetAssociatedMemberForXmlSyntax(CSharpSy
                 MemberDeclarationSyntax memberSyntax = curr as MemberDeclarationSyntax;
                 if (memberSyntax != null)
                 {
-                    // CONSIDER: require that the xml syntax precede the start of the member span?
+                    // Require that the xml syntax precedes the start of the member span.
+                    if (memberSyntax.SpanStart < xmlSyntax.SpanStart)
```

Comment:
>if (memberSyntax.SpanStart < xmlSyntax.SpanStart) [](http://example.com/codeflow?start=20&length=49)

Perhaps the check will be more robust an intuitive if we check whether ```structuredTrivia``` is among ```memberSyntax.GetLeadingTrivia()``` nodes. At least it looks like this is how ```GetDocumentationCommentTriviaFromSyntaxNode``` helper locates doc comments